### PR TITLE
installer: offer to configure `init.defaultBranch`

### PR DESCRIPTION
**Pull request by dscho**
Since v2.28.0, Git supports `init.defaultBranch` as a way to specify what branch name to use when initializing new repositories. Git for Windows does not have any UI to present that choice to the user. This PR changes that.

Running `./installer/release.sh -d DefaultBranch` in a local check out shows how the page looks like:
image

### CODE REVIEW

**Comment by dscho** [comment]
Through a private conversation, I got some useful feedback which suggested a better wording, which I used to update this PR: 
image

File: installer/install.iss [hunk 1]
```diff
@@ -1834,6 +1851,42 @@ begin
...
     EditorSelectionChanged(NIL);
 
+    (*
+     * Create a custom page for modifying the default branch.
+     *)
+
+    DefaultBranchPage:=CreatePage(PrevPageID,'Adjusting the name of the initial branch in new repositories','What would you like Git to call the default branch after "git init"?',TabOrder,Top,Left);
+
+    // 1st choice
+    RdbDefaultBranch[DB_Unspecified]:=CreateRadioButton(DefaultBranchPage,'Let Git decide','This leaves it to Git to name the initial branch in newly created repository (current default: "master").',TabOrder,Top,Left);
```
Comment (bbolli) [inline installer/install.iss:1861]: Missing line wrapping (see screen shot); the label is truncated.

**Review by bbolli — CHANGES_REQUESTED** [review]

**Comment by jeffhostetler** [comment]
Seems like we have "master" in the current default parenthetical on the first radio button and then "main" hinted in the text box under the second radio button.  I just worry that a user who is new to the whole controversy is going to wonder why this dialog is here and what is motivating this page.

Where you have "What would you like Git to call the default branch after `git init`?"  could you say something like:
"Historically, the default branch is called `master` following a `git init`; you can now change that!"

**Comment by dscho** [comment]
@bbolli @jeffhostetler Thank you for your valuable feedback. This is my current version:

image

**Comment by PhilipOakley** [comment]
LGTM. Thanks.

**Comment by jeffhostetler** [comment]
Looks good!  Yes, that helps motivate the dialog page.  Thanks!

**Review by jeffhostetler — APPROVED** [review]

File: installer/install.iss [hunk 2]
```diff
@@ -1834,6 +1858,50 @@ begin
...
     EditorSelectionChanged(NIL);
 
+    (*
+     * Create a custom page for modifying the default branch.
+     *)
+
+    DefaultBranchPage:=CreatePage(PrevPageID,'Adjusting the name of the initial branch in new repositories','What would you like Git to name the initial branch after "git init"?',TabOrder,Top,Left);
+
+    // 1st choice
+    RdbDefaultBranch[DB_Unspecified]:=CreateRadioButton(DefaultBranchPage,'Let Git decide','Let Git use its default branch name (currently: "master") for the initial branch'+#13+'in newly created repositories. The Git project <A HREF=https://sfconservancy.org/news/2020/jun/23/gitbranchname/>intends</A> to change this default to'+#13+'a more inclusive name in the near future.',TabOrder,Top,Left);
```
Comment (derrickstolee) [inline installer/install.iss:1868]: It looked like your `currently: "master")` was getting cut off in the picture. How can we make this part get word-wrapped?

Comment (derrickstolee) [reply installer/install.iss:1868]: Nevermind, you fixed that in a later picture.

**Review by derrickstolee — APPROVED** [review]

**Comment by dscho** [comment]
Okay, _slightly_ more succinct description for the second bullet point:
image

**Review by bbolli — APPROVED** [review]

**Comment by the-meme-factory** [comment]
Jesus, this is so fucking retarded

**Comment by PhilipOakley** [comment]
@the-meme-factory  hmm, civil language please.

**Pull request merged**